server/src/assembly/resources/conf/logback.xml

```diff
@@ -46,7 +46,7 @@
         <file>${IOTDB_HOME}/logs/log_warn.log</file>
         <rollingPolicy class="ch.qos.logback.core.rolling.SizeAndTimeBasedRollingPolicy">
             <fileNamePattern>${IOTDB_HOME}/logs/log-warn-%d{yyyyMMdd-HH}.%i.log.gz</fileNamePattern>
-            <maxFileSize>10MB</maxFileSize>
+            <maxFileSize>200MB</maxFileSize>
```

Comment:
If the value of maxfilesize is large and logs are queried and written during log compression, performance jitters may occur. This has been a problem in our production environment.

Reply:
emmm,What size do you think is appropriate

Reply:
Hi, @wangchao316 do you have any suggestions according to your experience?

Reply:
hi @neuyilan 
The log printing level and log printing frequency are distinguished.
Generally, the value of info is 35 MB.
Generally, the value of warn is 50 MB.
Generally, the debug value is 125 MB.

Reply:
Hi, @wangchao316 Thanks a lot.
Hi @yanhongwangg, maybe you could revise your codes according to @wangchao316's suggestions.